algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java

```diff
@@ -163,4 +162,9 @@ static boolean hasAgeGroupToInclude(@NotNull List<String> ageGroups) {
         }
         return false;
     }
+
+    @VisibleForTesting
+    static boolean hasPotentiallyOpenRequirementToInclude(@Nullable String locationStatus) {
```

Comment:
Is `locationStatus` a really descriptive name? In one place the method is called with `trial.recruitment()`, and the values look more like a recruitment status than the status of a location.

Reply:
indeed, I have update it to recruitmentStatus